src/screens/OrganizationActionItems/ItemDeleteModal.tsx

```
@@ -54,38 +56,36 @@ const ItemDeleteModal: React.FC<InterfaceItemDeleteModalProps> = ({
-      </Modal>
-    </>
+    <Modal className={styles.itemModal} onHide={hide} show={isOpen}>
+      <Modal.Header>
+        <p className={styles.titlemodal}>{t('deleteActionItem')}</p>
+        <Button
+          variant="danger"
+          onClick={hide}
+          className={styles.closeButton}
+          data-testid="modalCloseBtn"
+        >
+          <i className="fa fa-times"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)